src/Arcus.EventGrid.Proxy.Api/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM microsoft/dotnet:2.2-aspnetcore-runtime AS base
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.2-alpine AS base
```

Comment:
Was it like `...alpha10` ? I think the templates are using that.

Reply:
What do you mean? This is just specifying alpine for that version, not sure why we would use alpha?

Reply:
Oops, I meant `...alphine10`.

Reply:
That's ok, we don't care what alphine version we use